src/Invictus.Testing/LogicAppTriggerNotFoundException.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Security.Permissions;
+
+namespace Invictus.Testing
+{
+    /// <summary>
+    /// Exception thrown when no trigger can be found for a given logic app.
+    /// </summary>
+    [Serializable]
+    public class LogicAppTriggerNotFoundException : LogicAppException
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogicAppTriggerNotFoundException"/> class.
+        /// </summary>
+        public LogicAppTriggerNotFoundException()
```

Comment:
I would remove this overload, we need to have logic app information as a minimum